ocp-templates/app-interface/backend.yaml

```diff
@@ -0,0 +1,161 @@
+# Template for ibutsu prod server
+kind: Template
+apiVersion: template.openshift.io/v1
+metadata:
+  name: ibutsu-backend-template
+# ===============================================
+# Backend
+# ===============================================
+objects:
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    labels:
+      app: ${APP_NAME}
+    name: ibutsu-backend
+  spec:
+    replicas: 1
+    selector:
+      deploymentConfig: ibutsu-backend
+    strategy:
+      type: Rolling
+    template:
+      metadata:
+        labels:
+          app: ${APP_NAME}
+          deploymentConfig: ibutsu-backend
+      spec:
+        containers:
+        - env:
```

Comment:
I'm happy to do this as a follow-up, but there is one more secret env var we need to run Ibutsu + Auth:
https://github.com/ibutsu/ibutsu-server/issues/252